contracts/sfc/SFCLib.sol

```diff
@@ -651,6 +667,28 @@ contract SFCLib is SFCBase {
         return address(uint160(to));
     }
 
+    /// Convert an address to validator's address if the address is white-listed address.
+    /// Converts only to validators managed by Fantom Foundation. It is necessary to enable
+    /// smooth transition to the new sonic chain.
+    function _addrToValidator(address addr, uint256 validatorID) internal view returns(address) {
+        // Fantom's wallet to make requests from
+        if (addr != 0x0b2E90c831626A65a26f75153Be54aeaAeeb8363) {
+            return addr;
+        }
+
+        // Fantom's validators <1,11>, 64
+        if ((validatorID < 1 || validatorID > 11) && validatorID != 64) {
```

Comment:
`validatorID < 1` part is unnecessary - validator 0 cannot exist, it will return the same on the following check.